docs/apache-airflow/concepts/taskflow.rst

```diff
@@ -81,6 +81,49 @@ To use logging from your task functions, simply import and use Python's logging
 
 Every logging line created this way will be recorded in the task log.
 
+Passing Arbitrary Objects As Parameters
+---------------------------------------
+
+.. versionadded:: 2.5.0
+
+As mentioned TaskFlow uses XCom to pass variables to each task. This requires that variables that are used as parameters
+to tasks need to be able to be serialized. Airflow out of the box supports all built-in types (like int or str) and it
+supports objects that are decorated with ``@dataclass`` or ``@attr.define``, but it will need some help if you want to
+pass custom objects or if you want to control serialization yourself. To do so add the ``serialize()`` method to your
+class and the staticmethod ``deserialize(data: dict, version: int)`` to your class. Like so:
```

Comment:
```suggestion
TaskFlow uses XCom to pass variables to each task. This requires variables that are used as arguments
to tasks to be able to be serialized. Airflow out of the box supports all built-in types (like int or str) and it
supports objects that are decorated with ``@dataclass`` or ``@attr.define``, but it will need some help if you want to pass custom objects, or if you want to control serialization yourself. To do so, implement the ``serialize()`` method and ``deserialize(data: dict, version: int)`` staticmethod in your class. Like so:
```

Reply:
I *think* you are reformatting, but don't know exactly. Please reapply if you don't mind.

Reply:
Did some rewording for clarity